browse/commands/check_paper_formats.py

```diff
@@ -0,0 +1,61 @@
+"""Checks that the paper format in the DB matches what is in the FS.
+
+This script checks the arXiv_metadata.source_format plausibly matches the file extension of the source files."""
+from flask import  Blueprint
+from sqlalchemy import or_
+import json
+
+import click
+
+from browse.domain.identifier import Identifier
+from browse.services.database import db
+from browse.services.database.models import Metadata
+from browse.services.dissemination import get_article_store, ArticleStore
+from browse.services.documents.format_codes import formats_from_source_flag
+from browse.services.object_store import FileObj
+bp = Blueprint("check", __name__)
+
+@bp.cli.command("paper_formats")
+@click.argument("yymm")
+def check_paper_formats(yymm: str) -> None:
+    """Checks formats for yymm."""
+    query = (db.session.query(Metadata.paper_id, Metadata.version,
+                            Metadata.source_format,
+                            Metadata.source_flags, Metadata.source_size)
+                .filter(or_(Metadata.paper_id.like(f"%/{yymm}%"),
+                            Metadata.paper_id.like(f"{yymm}.%"))))
+
+    rows = query.all()
+    if not rows:
+        print(f"No rows found.")
+    a_store: ArticleStore = get_article_store()
+    results = {}
+    for row in rows:
+        paper_id, version, src_fmt, source_flags, source_size = row[0], row[1], row[2], row[3], row[4]
+        print(f"Doing {paper_id}v{version}")
```

Comment:
do you still want print statements?

Reply:
Yes, this package is a CLI for gathering some data about the different ways of making the format list for a paper. It is not intended to be used in a flask web app.